### PR TITLE
iOS-3377 Show toast on completion when moving blocks

### PR DESCRIPTION
**Pull request by joe-pusya**

### CODE REVIEW

**Review by ignatovv — APPROVED** [review]

**Review by mgolovko — APPROVED** [review]

File: Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageBlocksStateManager.swift [hunk 1]
```diff
@@ -291,12 +291,19 @@ final class EditorPageBlocksStateManager: EditorPageBlocksStateManagerProtocol {
                     guard let self, let id = targetDocument.children.last?.id,
                           let details = targetDocument.details else { return }
                     if !details.isList {
-                        try await move(position: .bottom, targetId: targetDocument.objectId, dropTargetId: id, blocksIds: filteredBlocksIds)
-                        toastPresenter.showObjectCompositeAlert(
```
Comment (mgolovko) [inline Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageBlocksStateManager.swift:295]: why `try await` syntaxis doesn't work? `toastPresenter` was called after move. `completion` called at the same time 🤔

Comment (joe-pusya) [reply Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageBlocksStateManager.swift:295]: it works, but there are some condition inside `move` that can go to return after check and we don't need to call `completion`

Comment (mgolovko) [reply Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageBlocksStateManager.swift:295]: Maybe add result reason, to use swift concurrency sintaxis? I expect the completion to be call every time

Comment (mgolovko) [reply Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageBlocksStateManager.swift:295]: or add specific error

Comment (joe-pusya) [reply Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageBlocksStateManager.swift:295]: I will add result 🙌

**Pull request merged**